src/compoz.ts

```diff
@@ -342,13 +380,22 @@ export class Compoz {
 		}
 
 		this.elInput.onkeydown = e => {
-			if ((e.ctrlKey || e.metaKey) && e.keyCode === this.keyCodeZ) {
-				document.execCommand("undo")
+			if (e.ctrlKey || e.metaKey) {
+				if (!e.shiftKey && e.keyCode === this.keyCodeZ) {
+					e.preventDefault()
+					this.undoContent()
+				}
+				if (e.shiftKey && e.keyCode === this.keyCodeZ) {
```

Comment:
I think we should add another common redo key for Windows's user.  As far as I can remember the key for undo in Windows is `CTRL+y` (or `CTRL+r`?).